ml-proto/spec/kernel.ml

```diff
@@ -82,9 +82,9 @@ and expr' =
   | Block of expr list                      (* execute in sequence *)
   | Loop of expr                            (* loop header *)
   | Break of var * expr option              (* break to n-th surrounding label *)
-  | Br_if of var * expr option * expr       (* conditional break *)
+  | Break_if of var * expr option * expr    (* conditional break *)
+  | Break_table of var list * expr option * expr  (* indexed break *)
```

Comment:
My point is many of the arguments there are also applicable here, both ways, and we couldn't resolve things there.